config.default.yaml

```diff
@@ -340,18 +340,35 @@ costs:
 
 
 monte_carlo:
+  # Description: Specify Monte Carlo sampling options for uncertainty analysis.
+  # Define the option list for Monte Carlo sampling.
+  # Make sure add_to_snakefile is set to true to enable Monte-Carlo
   options:
-    add_to_snakefile: false
-    samples: 7    # number of optimizations
-    sampling_strategy: "chaospy"    # "pydoe2", "chaospy", "scipy", packages that are supported
-  pypsa_standard:
-      # User can add here flexibly more features for the Monte-Carlo sampling.
-      # Given as "key: value" format
-      # Key: add below the pypsa object for the monte_carlo sampling, "network" is only allowed for filtering!
-      # Value: currently supported format [l_bound, u_bound] or empty [], represent multiplication factors for the object
-    loads_t.p_set: [0.9, 1.1]
-    # generators_t.p_max_pu.loc[:, n.generators.carrier == "wind"]: [0.9, 1.1]
-    # generators_t.p_max_pu.loc[:, n.generators.carrier == "solar"]: [0.9, 1.1]
+    add_to_snakefile: false # When set to true, enables Monte Carlo sampling
+    samples: 9 # number of optimizations. Note that number of samples when using scipy has to be the square of a prime number
+    sampling_strategy: "chaospy"  # "pydoe2", "chaospy", "scipy", packages that are supported
+    seed: 42 # set seedling for reproducibilty
+  # Different distributions can be specified for various PyPSA network object.
+  # Supported distributions for uncertainties are uniform, normal, lognormal, triangle, beta and gamma.
+  # More info on the distributions are documented in the Chaospy reference guide...
+  # https://chaospy.readthedocs.io/en/master/reference/distribution/index.html
+  # ... users can add flexibly more features for the Monte-Carlo sampling using the description below
+  # {pypsa network object, e.g. "loads_t.p_set"}:
+  # type: {any distribution among: "uniform", "normal", "lognormal", "triangle", "beta" and "gamma"}
+  # args: {arguments passed as a list depending on the distribution, see arguments description in Chaospy reference guide}
```

Comment:
There is the need to clarify the order of the arguments.
I see three options:
- add it into the readthedocs description and a crosslink here (if the docstring of the monte_carlo.py file si done well that works)
- add it here
- describe it into the examples below under uncertainties

Point 1 is the best; actually, it should be good to add the monte_carlo here into the API reference at least: https://github.com/pypsa-meets-earth/pypsa-earth/blob/main/doc/api_reference.rst

Here adding some description of at least few rules would be good too, at least doing point 3.
Feel free to propose something that looks compact to you :)

Reply:
@davide-f, Thanks for the suggestions. I have planned to include the instructions in the `readthedocs`